packages/mobile/src/geth/RNGethSigner.ts

```diff
@@ -0,0 +1,91 @@
+import { normalizeAddressWith0x } from '@celo/utils/src/address'
+import { Tx } from 'web3-core'
+import { Signer } from '@celo/contractkit/lib/wallets/signers/signer'
+import {
+  encodeTransaction,
+  extractSignature,
+  RLPEncodedTx,
+  rlpEncodedTx,
+} from '@celo/contractkit/lib/utils/signing-utils'
+import RNGeth from 'react-native-geth'
+import * as ethUtil from 'ethereumjs-util'
+
+const INCORRECT_PASSWORD_ERROR = 'could not decrypt key with given password'
+const currentTimeInSeconds = () => Math.round(Date.now() / 1000)
+
+/**
+ * Implements the signer interface on top of a RNGeth instance
+ */
+export class RNGethSigner implements Signer {
+  /**
+   * Construct a new instance of the RPC signer
+   *
+   * @param geth the RNGeth instance
+   * @param account Account address derived from the private key to be called in init
+   * @param unlockBufferSeconds Number of seconds to shrink the unlocked duration by to account for
+   * latency and timing inconsistencies on the node
+   * @param unlockTime Timestamp in seconds when the signer was last unlocked
+   * @param unlockDuration Number of seconds that the signer was last unlocked for
+   */
+  constructor(
+    protected geth: RNGeth,
+    protected account: string,
+    protected unlockBufferSeconds = 5,
+    protected unlockTime = -1,
+    protected unlockDuration = -1
+  ) {}
+
+  async init(privateKey: string, passphrase: string) {
+    return this.geth.addAccount(privateKey, passphrase)
+  }
+
+  async signRawTransaction(tx: Tx) {
+    if (normalizeAddressWith0x(tx.from! as string) !== this.account) {
+      throw new Error(`RNGethSigner(${this.account}) cannot sign tx with 'from' ${tx.from}`)
+    }
+    const encodedTx = rlpEncodedTx(tx)
+    const signature = await this.signTransaction(0, encodedTx)
+    return encodeTransaction(encodedTx, signature)
+  }
+
+  async signTransaction(
+    addToV: number,
```

Comment:
Can you add the comment below on addToV here to provide context on the parameter